sdks/python/apache_beam/coders/row_coder.py

```diff
@@ -183,14 +198,14 @@ def encode_to_stream(self, value, out, nested):
 
     self.NULL_MARKER_CODER.encode_to_stream(words.tobytes(), out, True)
 
-    for c, field, attr in zip(self.components, self.schema.fields, attrs):
-      if attr is None:
-        if not field.type.nullable:
+    for i in np.argsort(self.encoding_positions):
```

Comment:
Could you pre-compute np.argsort and store it (e.g. on `self.encoding_positions_argsort`) in `__init__` so we don't have to repeat it for each element? We could use the pre-computed argsort result here and in `decode_from_stream`

Reply:
That makes sense, I changed it.